collapse.js

```diff
@@ -0,0 +1,9 @@
+window.onload = function() {
+	let frames = document.getElementsByClassName('frame');
+	for (let i = 0; i < frames.length; i++) {
```

Comment:
this can be [for...in](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in)